pulp_ansible/tests/functional/api/collection/test_sync.py

```diff
@@ -7,7 +7,10 @@
 from pulp_ansible.tests.functional.constants import TEST_COLLECTION_CONFIGS
 from orionutils.generator import build_collection
 from pulpcore.client.pulp_ansible import PulpAnsibleArtifactsCollectionsV3Api
-from pulp_ansible.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
+from pulp_ansible.tests.functional.utils import (
+    monitor_task,
+    set_up_module as setUpModule,
+)  # noqa:F401
```

Comment:
I think this needs to be next to the setUpModule, even though I'm not sure what we use it for.

Reply:
After looking at noqas in pulpcore, I think it is supposed to go after the top parenthesis.